libraries/Microsoft.Bot.Builder.Dialogs/DialogContext.cs

```diff
@@ -354,7 +354,18 @@ public async Task<DialogTurnResult> EndDialogAsync(object result = null, Cancell
                     var dialog = this.FindDialog(ActiveDialog.Id);
                     if (dialog == null)
                     {
-                        throw new InvalidOperationException($"DialogContext.EndDialogAsync(): Can't resume previous dialog. A dialog with an id of '{ActiveDialog.Id}' wasn't found.");
+                        var rootDialog = this.DialogManager.RootDialog;
+
+                        try
+                        {
+                            return await this.ReplaceDialogAsync(rootDialog.Id, cancellationToken: cancellationToken).ConfigureAwait(false);
+                        }
+#pragma warning disable CS0168 // Variable is declared but never used
+                        catch (Exception err)
```

Comment:
If you are not using the exception, don't need declare a variable for it. Please remove the pragma.

Reply:
Thanks, removed!